examples/numbers/percent.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Displaying percentages
```

Comment:
This is currently not in the spec, but may be added soon. The concern is that we might add a separate function.